src/helpers.js

```diff
@@ -155,3 +156,6 @@ export const flex = ({
 
   return props;
 };
+
+// eslint-disable-next-line max-len
+export const stripBoolean = (Element) => ({ debug, fluid, fill, wrap, ...props }) => <Element {...props} />
```

Comment:
A small suggestion: Looping the props to form an object of bool and then de-structuring would be more future proof.

Reply:
export const stripBoolean = (Element) => ({...props}) => {
  for(let prop in props){ 
    (typeof props[prop] == 'boolean') && delete props[prop]; 
  }
  return <Element {...props} />
}